modal_proto/api.proto

```diff
@@ -2703,6 +2715,7 @@ service ModalClient {
   rpc EnvironmentUpdate(EnvironmentUpdateRequest) returns (EnvironmentListItem);
 
   // Functions
+  rpc FunctionAsyncUnary(FunctionAsyncUnaryRequest) returns (FunctionAsyncUnaryResponse);
```

Comment:
On second thought `Unary` can be a bit confusing re: gRPC meaning so maybe `Invoke`?